src/main/java/lotto/DAO/UpdatejdbcTemplate.java

```diff
@@ -0,0 +1,22 @@
+package lotto.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.List;
+
+public class UpdatejdbcTemplate {
```

Comment:
클래스 이름은 UpperCamelCase로 작성합니다. 아래의 Google Java Style Guide를 참고해 주세요.
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

```suggestion
public class UpdateJdbcTemplate {
```